### PR TITLE
Document how to load style settings into IDEA and Eclipse

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Per the comment in this [PR](https://github.com/informatici/openhospital-core/pull/149) here are some quick and dirty instructions as to how to use the settings files for IntelliJ IDEA and Eclipse.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,37 @@ If you want to rebuild local db you should follow these steps:
...
+* Select *Settings* in the *File* menu
+* Select *Editor*
+* Select *Code Style*
+* Expand the menu item and select *Java*
+* Go to *Scheme* at the top, click on the setting button by the side of the drop-down list
+* Select *Import Scheme*
+* Select *IntelliJ IDE code style XML*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/idea/OpenHospital-code-style-configuration.xml`
+* Select *OK* 
+* At this point the code style is stored as part of the IDE and is used for **all** projects opened in the editor.  To restrict the settings to just this project again select the setting button by the side of the *Scheme* list and select *Copy to Project...*. If successful a notice appears in the window that reads: *For current project*.
```
Comment (mwithi) [inline README.md:54]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>IntelliJ instructions</summary>

* list1
* list2

</details>

File: README.md [hunk 2]
```diff
@@ -35,3 +35,37 @@ If you want to rebuild local db you should follow these steps:
...
+* Select *Formatter*
+* Select the *Import...* button
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital-Java-CodeStyle-Formatter.xml`
+* Select *Open*
+* At this point the code style is stored and is applicable to all projects opened in the IDE.  To restrict the settings just to this project select *Configure Project Specific Settings...* in the upper right.  In the next dialog select the *openhospital* repository and select *OK*.  In the next dialog select the *Enable project specific settings* checkbox.  Finally select *Apply and Close*.
+* Back in the *Code Style* menu area, select *Organize Imports*
+* Select *Import...*
+* Navigate to the location of the file which relative to the project root is:  `.ide-settings/eclipse/OpenHospital.importorder`
+* Select *Open*
+* As with the formatting styles the import order is applicable to all projects.  In order to change it just for this project repeat the same steps as above for *Configure Project Specific Settings...*
```
Comment (mwithi) [inline README.md:70]: I would suggest a collapsable section for the list, like this:

<details>
  <summary>Eclipse instructions</summary>

* list1
* list2

</details>

**Review by mwithi — APPROVED** [review]
LGTM, if you like you may apply the suggested changes (see comments)

File: README.md (hunk 1 above)
Comment (dbmalkovsky) [reply README.md:56]: Done for both

**Pull request merged**